kmk/modules/midi.py

```diff
@@ -0,0 +1,108 @@
+# Originally put together by xs5871 on the KMK Firmware Discord/Matrix channel
```

Comment:
Thanks for mention, but that's not necessary. It's not like we're linking to another resource for reference.